helper-cli/src/main/kotlin/commands/VerifySourceArtifactCurationsCommand.kt

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
```

Comment:
Nit: I believe we decided to always use "2017 -" even for newly added files (like you also did in your new file in the Postgres PR) to more easily search & replace copyright years in new years.

Reply:
As discussed in the chat, will be aligned separately.